content/docs/configuration/vault.md

```diff
@@ -254,6 +256,101 @@ Kubernetes 1.24 and above.
               key: token
     ```
 
+### Authenticating with a Kubernetes Service Account Without A Secret
+
+The [previous method](#static-service-account-token) of authenticating with
+Vault has a major disadvantage: it relies on a less secure "static" token (by
+"static", we mean a token that does not have an expiry time).
+
+Using the field `serviceAccountRef` instead of `secretRef`, you can let
+cert-manager request ephemeral tokens.
```

Comment:
I think you should elaborate more on what it means to "request ephemeral tokens".
For example: https://cert-manager.io/docs/configuration/acme/dns01/azuredns/#managed-identity-using-aad-workload-identity

Reply:
I added a paragraph on "request ephemeral tokens". Hopefully it makes more sense now.